unit/src/HitTests.C

```diff
@@ -1,626 +0,0 @@
-      {"no escaped newline",
-       "foo='hello\\nworld'",
-       "foo",
-       "hello\\nworld",
-       hit::Field::Kind::String},
-      {"cosecutive string literal 1", "foo='bar''baz'", "foo", "barbaz", hit::Field::Kind::String},
-      {"cosecutive string literal 2",
-       "foo='bar'\n\n'baz'",
-       "foo",
-       "barbaz",
-       hit::Field::Kind::String},
-      {"path-normalize-find #12313",
-       "[foo][/bar]baz=42[][]",
-       "foo/bar/baz",
-       "42",
-       hit::Field::Kind::Int},
-
-  };
-
-  for (size_t i = 0; i < sizeof(cases) / sizeof(ValCase); i++)
-  {
-    auto test = cases[i];
-    auto root = hit::parse("TEST", test.input);
-    hit::BraceExpander exw("TEST");
-    root->walk(&exw);
-    auto n = root->find(test.key);
-    if (!n)
-    {
-      FAIL() << "case " << i + 1 << " (" << test.name << ") failed to find key '" << test.key
-             << "'\n";
-      continue;
-    }
-    if (n->strVal() != test.val)
-    {
-      FAIL() << "case " << i + 1 << " (" << test.name << ") wrong value (key=" << test.key
-             << "): got '" << n->strVal() << "', want '" << test.val << "'\n";
-      continue;
-    }
-
-    auto f = dynamic_cast<hit::Field *>(n);
-    if (!f)
-      FAIL() << "case " << i + 1 << " node type is not NodeType::Field";
-    else if (f->kind() != test.kind)
-      FAIL() << "case " << i + 1 << " wrong kind (key=" << test.key << "): got '"
-             << strkind(f->kind()) << "', want '" << strkind(test.kind) << "'\n";
-  }
-}
-TEST(HitTests, BraceExpressions)
-{
-  ValCase cases[] = {
-      {"substitute string", "foo=bar boo=${foo}", "boo", "bar", hit::Field::Kind::String},
-      {"substitute string explicit",
-       "foo=bar boo=${replace foo}",
-       "boo",
-       "bar",
-       hit::Field::Kind::String},
-      {"trailing space", "foo=bar boo=${foo} ", "boo", "bar", hit::Field::Kind::String},
-      {"substute number", "foo=42 boo=${foo}", "boo", "42", hit::Field::Kind::Int},
-      {"multiple replacements",
-       "foo=42 boo='${foo} ${foo}'",
-       "boo",
-       "42 42",
-       hit::Field::Kind::String},
-      {"nested", "foo=bar [hello] boo='${foo}' []", "hello/boo", "bar", hit::Field::Kind::String},
-      {"repl-header-before",
-       "src=foo [src] bar='${src}' []",
-       "src/bar",
-       "foo",
-       hit::Field::Kind::String},
-      {"repl-header-missing", "[src] bar='${src}' []", "src/bar", "${src}", hit::Field::Kind::None},
-      {"repl-header-shadow",
-       "[src] bar='${src}' [] src=foo",
-       "src/bar",
-       "${src}",
-       hit::Field::Kind::None},
-      {"nested shadow",
-       "foo=bar [hello] foo=baz boo='${foo}' []",
-       "hello/boo",
-       "baz",
-       hit::Field::Kind::String},
-      {"multi-line brace expression",
-       "foo=${raw 4\n"
-       "          2\n"
-       "     }",
-       "foo",
-       "42",
-       hit::Field::Kind::String},
-      {"fparse", "foo=${fparse 40 + 2}\n", "foo", "42", hit::Field::Kind::Float},
```

Comment:
Any test with fparse needs to stay, this feature is not tested in the moosetools repository because fparser is not included. All the others can be removed.